plugins/woocommerce-admin/client/marketplace/components/product-list/types.ts

```diff
@@ -35,6 +36,10 @@ export interface Product {
 	productType?: string;
 	averageRating?: number | null;
 	reviewsCount?: number | null;
+	label?: string;
```

Comment:
Not a big deal, but it would be nice to not add this data to the product as it's not related.
A separate object for tracking data would be ideal.

Reply:
Agreed, moved them to a different prop and type: https://github.com/woocommerce/woocommerce/pull/41365/commits/a9f85449d20736dbc78723e2e054bd8d434d894e